packages/astro/src/runtime.ts

```diff
@@ -396,7 +396,7 @@ async function createSnowpack(astroConfig: AstroConfig, options: CreateSnowpackO
           config: {
             plugins: {
               [resolveDependency('autoprefixer')]: {},
-              ...(astroConfig.devOptions.tailwindConfig ? { [resolveDependency('tailwindcss')]: {} } : {}),
+              ...(astroConfig.devOptions.tailwindConfig ? { [resolveDependency('tailwindcss')]: astroConfig.devOptions.tailwindConfig } : {}),
```

Comment:
This is expecting an object, but `tailwindConfig` is a string. I think we need to import this file instead?

Reply:
See https://github.com/snowpackjs/astro/pull/352#discussion_r649983084